azurerm/resource_arm_mariadb_server.go

```diff
@@ -123,6 +123,7 @@ func resourceArmMariaDbServer() *schema.Resource {
 				ForceNew: true,
 				ValidateFunc: validation.StringInSlice([]string{
 					"10.2",
+					"10.3",
```

Comment:
can we add this [to the documentation too](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/mariadb_server.html.markdown#L66)? we'd generally phrase that as:

```
* `version` - (Required) Specifies the version of MariaDB to use. Possible values are `10.2` and `10.3`. Changing this forces a new resource to be created.
```